app/formats/supertypes.yml

```diff
@@ -17,5 +17,10 @@
 
 - id: consultations
   label: Consultations
-  description: To request users views or evidence on an issue and to build collective
-    agreement.
+  description: To request users views or evidence on an issue and to build collective agreement.
+
+- id: not-sure
+  label: I'm not sure this should be on GOV.UK
+  description: View this guide to what should go on GOV.UK and where else we can publish content.
```

Comment:
I know this copied verbatim but is it missing a word "View this guide to _see_ what should" perhaps? and maybe "where else _~~we~~you_ can publish content"

Reply:
Good spot - corrected.